pkg/dsync/drwmutex.go

```diff
@@ -454,12 +464,9 @@ func (dm *DRWMutex) RUnlock() {
 	// Tolerance is not set, defaults to half of the locker clients.
 	tolerance := len(restClnts) / 2
```

Comment:
Shouldn't tolerance be len(restClnts)/2 + 1 here ?

Reply:
No, it won't be @vadmeste `tolerance` is not quorum - tolerance allowed failure rate.

Reply:
> No, it won't be @vadmeste `tolerance` is not quorum - tolerance allowed failure rate.

yes I see now, we are mainly relying on checkFailedUnlocks() to check if N/2+1 suceed to unlock or not